client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -204,30 +204,30 @@ const MobileNav = () => {
   const project = useSelector((state) => state.project);
   const user = useSelector((state) => state.user);
 
-  const { t } = useTranslation();
+  // const { t } = useTranslation();
 
   const editorLink = useSelector(selectSketchPath);
-  const pageName = useWhatPage();
+  const { pageName, title } = useWhatPage();
 
   // TODO: remove the switch and use a props like mobileTitle <Nav layout=“dashboard” mobileTitle={t(‘Login’)} />
-  function resolveTitle() {
-    switch (pageName) {
-      case 'login':
-        return t('LoginView.Login');
-      case 'signup':
-        return t('LoginView.SignUp');
-      case 'account':
-        return t('AccountView.Settings');
-      case 'examples':
-        return t('Nav.File.Examples');
-      case 'myStuff':
-        return 'My Stuff';
-      default:
-        return project.name;
-    }
-  }
-
-  const title = useMemo(resolveTitle, [pageName, project.name]);
+  // function resolveTitle() {
```

Comment:
Although we can't get this in, I wanted to leave a few notes for your work! 

Ideally, we want to ensure that comments provide supportive information. Since this function is being removed, ideally lines `212` and  `213 - 230` could simply be deleted. Since this issue is attempting to address the TODO, line `212` could be removed as well.